src/components/post/PostsComponent.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+    Link
+  } from "react-router-dom";
+
+export default function PostComponent({posts}:{posts:any}) {
```

Comment:
make a folder called `src/types` or `src/models` and define new type for `posts`